diffeqpy/tests/test_dae.py

```diff
@@ -1,9 +1,7 @@
-import diffeqpy
+from .. import de
```

Comment:
what's this?

Reply:
It's Python's relative import syntax. `from .. import de` is equivalent to `from diffeqpy import de` here. I just use it whenever I can, since it's easier to rename the package, for example.